.circleci/config.yml

```diff
@@ -129,7 +129,18 @@ jobs:
     steps:
       - attach_workspace: *attach_options
       - run: yarn webdriver-update
-      - run: yarn test-large -- --full --nb-shards=${CIRCLE_NODE_TOTAL} --shard=${CIRCLE_NODE_INDEX}
+      - run: yarn test-large --full --nb-shards=${CIRCLE_NODE_TOTAL} --shard=${CIRCLE_NODE_INDEX}
+
+  test-large-ivy:
+    <<: *defaults
+    docker:
+      - image: *browsers_docker_image
+    resource_class: large
+    parallelism: 4
+    steps:
+      - attach_workspace: *attach_options
+      - run: yarn webdriver-update
+      - run: yarn test-large --ivy --full --glob="packages/angular_devkit/build_angular/test/{browser,karma,server}/*_spec_large.ts" --nb-shards=${CIRCLE_NODE_TOTAL} --shard=${CIRCLE_NODE_INDEX}
```

Comment:
I don't think we should add `karma` yet as `ng test` doesn't use ivy yet. https://github.com/angular/angular-cli/issues/14547